src/client/content_cao.cpp

```diff
@@ -1678,6 +1680,7 @@ void GenericCAO::processMessage(const std::string &data)
 		v3f position = readV3F32(is);
 		v3f rotation = readV3F32(is);
 		m_bone_position[bone] = core::vector2d<v3f>(position, rotation);
+		m_stop_animations = readU8(is);
```

Comment:
Protocol bump? Doesn't look backwards compatible.

Reply:
If you want the default to be `0`, this code is be fine.
If not, then some backwards compatibility code would be needed (no protocol bump).